extra/docker/develop/Dockerfile

```diff
@@ -18,20 +17,20 @@ ENV PATH="/home/pwntools/.local/bin:${PATH}"
 
 # Install Pwntools to the home directory, make it an editable install
 RUN git clone https://github.com/Gallopsled/pwntools \
- && python  -m pip install --upgrade --editable pwntools \
  && python3 -m pip install --upgrade --editable pwntools \
  && PWNLIB_NOTERM=1 pwn version
 
 # Requirements for running the tests
-RUN python  -m pip install --upgrade --requirement pwntools/docs/requirements.txt \
- && python3 -m pip install --upgrade --requirement pwntools/docs/requirements.txt
+RUN python3 -m pip install --upgrade --requirement pwntools/docs/requirements.txt
 
 # Python niceties for debugging
-RUN python  -m pip install -U ipython ipdb \
- && python3 -m pip install -U ipython ipdb
+RUN python3 -m pip install -U ipython ipdb
```

Comment:
While at it, could these `pip install`s be moved after the `apt install`s? It looks like we might install several packages twice. It should be in a single RUN call preferably, too.  We should also remove the apt cache after `apt install` to take advantage of Docker layer caching (here and in base).

Reply:
OK